hw2/04-background.js

```diff
@@ -0,0 +1,27 @@
+let timeInterval = setInterval(function () {
+  let randomColor1 = Math.floor(Math.random() * 255);
+  document.body.style.backgroundColor = "#" + randomColor1;
+}, 3000);
+let btnSelector = document.querySelector(".btn");
+btnSelector.addEventListener("click", changeBackgroundColor);
+function changeBackgroundColor(event) {
+  clearInterval(timeInterval);
+  event.preventDefault();
+  event.stopPropagation();
```

Comment:
What is the main purpose of this line? I am asking so that I can understand the implementation decision a bit better.